README.md

```diff
@@ -1,26 +1,32 @@
 # README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
 * Ruby version
    - 3.0.0p0 
 
-* System dependencies
-   - 
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+About the project: 
+Modern day science emphasizes the great importance of healthy and fulfilling interactions with food. Our quality of life and mental health is dependent upon on the daily food choices we make. Here at MegaBites we wish to empower you to make the choices that are right for you. We wanted to create an app where you can search recipes consistent with the top 4 diets backed by science and save any recipe you like to your own user recipe dashboard.   
+
+Built With:
+- React https://reactjs.org/docs/getting-started.html
+- Ruby on Rails https://guides.rubyonrails.org/
```

Comment:
For major libraries or frameworks like React and Ruby on Rails, it'd be good to indicate versions so that possible contributors know right away